modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/validation/PojosModelValidatorTestCase.java

```diff
@@ -61,6 +61,14 @@ public void pojoImplementsEqualsAndHashCode() {
     validate(ValidTestConnector.class);
   }
 
+  @Test
+  public void pojoConfigurationMustHaveDefaultConstructor() {
+    expectedException.expect(IllegalModelDefinitionException.class);
+    expectedException.expectMessage("Type 'InvalidPojoRequiresDefaultConstructor' does not have a default constructor");
+    validate(InvalidConfigParameterTestConnector.class);
+  }
+
+
```

Comment:
Add a failing test that checks the amount of errors with an extension that has a config with 2 pojos, that only one has a default constructor. Amount of errors should be 1.